src/service/jwt.service.js

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+import { ConsoleLogger } from '../untils/consoleLogger.js';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
```

Comment:
Ensure that the `user` object passed to `jwt.sign` does not contain sensitive information, as the payload of a JWT can be decoded by anyone with access to the token.